packages/core-state/src/wallets/wallet-manager.ts

```diff
@@ -252,104 +252,45 @@ export class WalletManager implements State.IWalletManager {
 
     public async applyBlock(block: Interfaces.IBlock): Promise<void> {
         this.currentBlock = block;
-        const generatorPublicKey: string = block.data.generatorPublicKey;
-
-        let delegate: State.IWallet;
-        if (!this.has(generatorPublicKey)) {
-            const generator: string = Identities.Address.fromPublicKey(generatorPublicKey);
-
-            if (block.data.height === 1) {
-                delegate = new Wallet(generator);
-                delegate.publicKey = generatorPublicKey;
-
-                this.reindex(delegate);
-            } else {
-                app.forceExit(`Failed to lookup generator '${generatorPublicKey}' of block '${block.data.id}'.`);
-            }
```

Comment:
It appears that this shortcut automatically creating wallet for the first block wasn't used